app/views/admin/admin_show.php

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><?= htmlspecialchars($post->title); ?> - Admin</title>
```

Comment:
I'd suggest we santise the variables to be output in the Model class, rather than in this view

Reply:
On second thoughts, I think your approach here is ok.  I think there's some case for doing this escaping in the view. I'm happy to leave this as is

Reply:
Good advice. Just to be consistent, for the backend, need to focus on the core logic which also includes sanitizing the data, for the front-end, just focus on rendering the clean data. I do need to refactor my model as well as my controller to achieve that.